evaluator/evaluate.go

```diff
@@ -548,6 +560,8 @@ func evaluateHandler(event mqueue.PlatformEvent) error {
 func loadCache() {
 	memoryPackageCache = NewPackageCache(enablePackageCache, preloadPackageCache, packageCacheSize, packageNameCacheSize)
 	memoryPackageCache.Load()
+	memoryVmaasCache = *NewVmaasPackageCache(enableVmaasCache, vmaasCacheSize, vmaasCacheCheckDuration)
```

Comment:
why are you derefernecing VmaasCache? Is there some downside with having the global var `memoryVmaasCache` of type `VmaasCache`?